src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -138,6 +153,7 @@ class CoronaTestService {
 		getRegistrationToken(
 			forKey: teleTAN,
 			withType: "TELETAN",
+			dateOfBirthKey: nil,
```

Comment:
is it correct that a PCR test by TeleTan has no date of birth?

Reply:
Yes, there are no test certificates for TeleTAN PCR tests. We don't have the data required for the certificate in case of TeleTANs. And they are always positive, so there is no need for a certificate which only make sense for negative tests.